### ISSUE TITLE
Wrong path on [ERROR rust_analyzer::main_loop] flycheck 0: ...

### ISSUE THREAD

**Issue by Miezhiko** [rust-lang/rust-analyzer#14786]
I need help with `[ERROR rust_analyzer::main_loop] flycheck 0: File with cargo diagnostic not found in VFS: file not found: WRONG PATH HERE`

using vscode (but not with rustup), still I can't understand where is the path is coming from here
but it's not from `"rust-analyzer.cargo.sysrootSrc"`

I need to find place where is it coming from and change path there. Path is pointing sandbox which was used to build rust actually.

**Comment by lnicola** [comment]
Do you have an extra `settings.json` around with a `linkedProjects` entry?

**Comment by Miezhiko** [comment]
> Do you have an extra `settings.json` around with a `linkedProjects` entry?

no, but I was trying to set sysroot and sysrootSrc, it wasn't helping

**Comment by dr-kernel** [comment]
I'm having the same problem, but with NeoVIM.  The path is correct, the file does exist but the error is getting thrown and its crashing RA.
